src/openapi/transport/core.ts

```diff
@@ -212,7 +214,6 @@ class Transport {
      *         }
      * });
      */
-    put = this.generateTransportCall('PUT');
 
     /**

```

Comment:
Those comments are now left alone. Do those comments apply to all subclasses if yes then we can move them to the base class otherwise I'm not sure what to do with them WDYT?

Reply:
those comments mainly talk about transport code, so I thought to keep in then and there, and once we ll migrate those to better documentation we'll pick from the respective file.
IMO they can not be move to base class